ms1/matching.py

```diff
@@ -331,26 +367,27 @@ def data_analysis(raw_data_df: pd.DataFrame, theo_masses_df: pd.DataFrame, rt_wi
     cleaned_data_df = clean_up(cleaned_df, sugar, time_delta_window)
 
     cleaned_data_df.sort_values('inferredStructure', inplace=True, ascending=True)
-    print(master_list.shape)
+
     return cleaned_data_df
-    
-    # cleaned_data_df.to_csv(ftrs_filepath[:-5] + '(2x DeAc)' + '.csv', index=False)
-    # # cleaned_data_df.to_excel(mq_filepath + ' Matched' + '.xlsx' , index=False)
-    # print(ftrs_filepath)
-    # #Raw matched data for debugging
-    # # ff.sort_values('inferredStructure', inplace=True, ascending=True)
-    # # ff.to_csv(ftrs_filePath + 'matched' + '.csv', index=False)
+
+
+def dataframe_to_csv(save_filepath: str, filename:str ,output_dataframe: pd.DataFrame ):
+    write_location = save_filepath + '/' + filename + '.csv'
+    output_dataframe.to_csv(write_location, index=False)
 
 
 if __name__== "__main__":
 
-    ftrs_filepath = r"C:\Users\Hyperion\Documents\GitHub\Mass-Spec-MS1-Analysis\data\test_ms_data.ftrs"
-    # mq_filepath = r"G:\Shared drives\MS1 Paper shared drive\Maxquant settings tests\Pseudomonas B1 T1 1K intensity Threshold.xlsx"
+    ftrs_filepath = r"G:\My Drive\PeptidoGlycan_DATA\2021-02 Pasteur (D. Lyras C. diff beta-eliminated)\FTRS files\20210129_Cdiff_betaL_SM1.ftrs"
```

Comment:
I'd consider getting rid of this and it's associated hardcoded vars, and using demo.py or an untracked equivalent.

Reply:
Maybe our test and demo code should move to using txt not ftrs?

Reply:
Yeah, that would be best. I've put maxquant data which is a text file in the data folder so I'll make changes so that it references this data. I'll have to check the baseline for the MQ data first before we switch from FTRS to maxquant files.